### PR TITLE
[WIP] added all([...effects]) to yield [...effects] and updated redux-saga to 0.15.3

### PR DESCRIPTION
**Pull request by dervos**
`yield [...effects]` is deprecated, use `yield all([...effects])`

### CODE REVIEW

File: src/sideEffect/saga/crudResponse.js [hunk 1]
```diff
@@ -23,25 +23,25 @@ import linkToRecord from '../../util/linkToRecord';
 function* handleResponse({ type, requestPayload, error, payload }) {
     switch (type) {
     case CRUD_UPDATE_SUCCESS:
-        return requestPayload.redirect ? yield [
+        return requestPayload.redirect ? yield all([
             put(showNotification('aor.notification.updated')),
             put(push(requestPayload.basePath)),
-        ] : yield [put(showNotification('aor.notification.updated'))];
+        ]) : yield all([put(showNotification('aor.notification.updated'))]);
```
Comment (JulienDemangeon) [inline src/sideEffect/saga/crudResponse.js]: The all effect is useless here, yield put() without the array should suffice

File: src/sideEffect/saga/crudResponse.js [hunk 2]
```diff
@@ -23,25 +23,25 @@ import linkToRecord from '../../util/linkToRecord';
...
     case CRUD_CREATE_SUCCESS:
-        return requestPayload.redirect ? yield [
+        return requestPayload.redirect ? yield all([
             put(showNotification('aor.notification.created')),
             put(push(linkToRecord(requestPayload.basePath, payload.data.id))),
-        ] : yield [put(showNotification('aor.notification.created'))];
+        ]) : yield all([put(showNotification('aor.notification.created'))]);
```
Comment (JulienDemangeon) [inline src/sideEffect/saga/crudResponse.js]: same

File: src/sideEffect/saga/crudResponse.js [hunk 3]
```diff
@@ -23,25 +23,25 @@ import linkToRecord from '../../util/linkToRecord';
...
     case CRUD_DELETE_SUCCESS:
-        return requestPayload.redirect ? yield [
+        return requestPayload.redirect ? yield all([
             put(showNotification('aor.notification.deleted')),
             put(push(requestPayload.basePath)),
-        ] : yield [put(showNotification('aor.notification.deleted'))];
+        ]) : yield all([put(showNotification('aor.notification.deleted'))]);
```
Comment (JulienDemangeon) [inline src/sideEffect/saga/crudResponse.js]: same

File: src/sideEffect/saga/crudResponse.js [hunk 4]
```diff
@@ -52,12 +52,12 @@ function* handleResponse({ type, requestPayload, error, payload }) {
         const errorMessage = typeof error === 'string'
             ? error
             : (error.message || 'aor.notification.http_error');
-        return yield [
+        return yield all([
```
Comment (JulienDemangeon) [inline src/sideEffect/saga/crudResponse.js]: same

File: src/sideEffect/saga/crudResponse.js (hunk 1 above)
Comment (dervos) [inline src/sideEffect/saga/crudResponse.js]: I will remove the useless effect and leave the `yield put()`.

**Comment by dervos** [comment]
Will pull against next. I would consider it merge-able if the e2e tests pass.

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**